src/app/navigation/HomeTabsNavigationStack.tsx

```diff
@@ -38,9 +38,15 @@ export type HomeTabsScreens = {
   Artwork: { slug: string; contextArtworkSlugs?: string[] }
   AlbumArtworks: { albumId: string }
   ShowTabs: { slug: string }
-  AddArtworkToAlbum: { slug: string }
+  AddArtworkToAlbum: { slug: string; contextArtworkSlugs?: string[] }
   ArtworkWebView: { uri: string }
-  CreateOrEditAlbum: { mode: "create" | "edit"; albumId?: string }
+  CreateOrEditAlbum: {
+    mode: "create" | "edit"
+    albumId?: string
+    artworkFromArtistTab?: string
+    slug?: string
+    contextArtworkSlugs?: string[]
```

Comment:
```suggestion
    navBackScreen: string
    navBackParams: {}
```

i would much prefer this to be more general like this, but I am currently struggling to think of another screen that we might need to navigate back too, so I am tempted to leave this as is.

the rest of the PR is 👍, so ill approve, assign it to you, and you decide if we leave it as is or change this, since you have more context about this. merge when you want 👍.

Reply:
I agree, but I think for now we can leave it as is, if we get more usage of that screen later then we can use the general one you suggested :)